app/src/main/res/values/strings.xml

```diff
@@ -322,17 +322,26 @@
   <string name="displayseed_title">Wallet recovery phrase</string>
   <string name="displayseed_authenticate_button">Display seed</string>
   <string name="displayseed_loading">Unlocking seed…</string>
-  <string name="displayseed_error">Seed could not be read. Please try again.\n\nReason: %1$s</string>
+  <string name="displayseed_error_generic">Could not unlock seed.\nPlease try again.</string>
+  <string name="displayseed_error_wrong_password">PIN is incorrect.</string>
   <string name="displayseed_instructions"><![CDATA[
-    This list of words is your wallet backup. Save it somewhere safe (not on this phone)!
+    The seed of the wallet is a list of 12 english words. It is your wallet backup. It gives full access to the funds in the wallet.
     <br /><br />
-    <b>Do not share it with anyone.</b>
-    <b>Do not lose it.</b> If you lose your words list and your phone, you\'ve lost your funds.
+    <strong>Only you alone possess this seed. Keep it private.</strong>
+    <br /><br />
+    <strong>Do not share this seed with anyone</strong>.
+    <br />
+    Beware of phishing. The developers of Phoenix will never ask for your seed.
+    <br /><br />
+    <strong>Do not lose this seed</strong>.
+    <br />
+    Save it somewhere safe (not on this phone). If you lose your seed and your phone, you\'ve lost your funds.
   ]]></string>
   <string name="displayseed_derivation_path">BIP39 seed with standard BIP84 derivation path</string>
   <string name="displayseed_words_td"><![CDATA[<td><font color="rgba(100,100,100,.3)"><small>#%1$d</small></font> <b>%2$s</b></td>]]></string>
-  <string name="displayseed_has_saved_seed_checkbox">I have made a backup of my seed</string>
+  <string name="displayseed_has_saved_seed_checkbox">I have made a backup of my seed, for example on a piece of paper.</string>
```

Comment:
Translation to Portuguese Brazilian: Fiz o backup da minha seed, por exemplo, em um folha de papel.

Reply:
fixed with 53d9c35